### PR TITLE
isVisible content update

### PR DESCRIPTION
**Pull request by vinodhrajamuralidharan**
isVisible content update

https://github.com/ember-learn/deprecation-app/issues/535

### CODE REVIEW

File: content/ember/v3/component-is-visible.md [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+      return A(['john', 'peter']);
+    }
+```
+{{! app/templates/components/my-question.hbs }}
+{{! `if` helper }}
+{{#if this.students.length}}
   <MyComponent />
 {{/if}}
+```
+or in the html element, we can use [`hidden`] (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute and pass the value dynamically:
```
Comment (jessica-jordan) [inline content/ember/v3/component-is-visible.md]: I think it'd be
```suggestion
or in the html element, we can use the [`hidden`] (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute and pass the value dynamically:
```

File: content/ember/v3/component-is-visible.md [hunk 2]
```diff
@@ -1,21 +1,48 @@
...
+  {{! app/templates/components/my-question.js }}
+  get hideMe() {
+    return true;
+  }
 ```
 
+```hbs
+  {{! app/templates/components/my-question.hbs }}
+  {{! `hidden` attribute }}
+  <div hidden={{this.hideMe}}></div>
```
Comment (jessica-jordan) [inline content/ember/v3/component-is-visible.md]: Nice example, by the way :sparkles:

File: content/ember/v3/component-is-visible.md [hunk 3]
```diff
@@ -1,21 +1,48 @@
...
+So, we were using something like this,
+```js
+App.MyView = Ember.View.extend({
+  isVisible: Ember.computed.alias('controller.myViewVisible')
+}
+```
+
+or in the component
+
+```js
```
Comment (jessica-jordan) [inline content/ember/v3/component-is-visible.md]: Would it make sense to add the context of the component into this example as well? f.e. by adding the import and export statements for the component class - what do you think?

Comment (vinodhrajamuralidharan) [reply content/ember/v3/component-is-visible.md]: can you check now?

Comment (jessica-jordan) [reply content/ember/v3/component-is-visible.md]: This sounds great!

**Comment by vinodhrajamuralidharan** [comment]
@jessica-jordan assign me some tasks like code, documentation, infrastructure, maintenance, triaging. I will do whenever I have free time.

**Pull request merged**